ros/processor/insights_engine_consumer.py

```diff
@@ -93,10 +94,11 @@ def handle_msg(self, msg):
                 host['id'],
                 custom_prefix=self.prefix
             )
-            reports = msg["results"]["reports"]  \
-                if msg["results"]["reports"] \
-                and type(msg["results"]["reports"]) == list \
-                else []
+            if msg["results"]["reports"] \
+                    and type(msg["results"]["reports"]) == list:
+                reports = msg["results"]["reports"]
+            else:
+                reports = []
```

Comment:
Nitpick - instead of `if..else` loop, does it make sense to initialize `reports` to empty array?

```suggestion
            reports = []
            if msg["results"]["reports"] \
                    and type(msg["results"]["reports"]) == list:
                reports = msg["results"]["reports"]
```